etl-data-pipeline/src/main/java/gov/cdc/etldatapipeline/changedata/model/dto/PatientOP.java

```diff
@@ -0,0 +1,141 @@
+package gov.cdc.etldatapipeline.changedata.model.dto;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import gov.cdc.etldatapipeline.changedata.model.odse.DebeziumMetadata;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import lombok.*;
+
+@Data
+@Entity
+@AllArgsConstructor
+@NoArgsConstructor
+@ToString
+@EqualsAndHashCode
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class PatientOP extends DebeziumMetadata {
+    @Id
+    @Column(name = "PATIENT_UID")
+    private String patientUid;
+    @Column(name = "PATIENT_MPR_UID")
+    private String patientMprUid;
+    @Column(name = "PATIENT_GENERAL_COMMENTS")
+    private String patientGeneralComments;
+    @Column(name = "PATIENT_ADD_TIME")
+    private String patientAddTime;
+    @Column(name = "PATIENT_RECORD_STATUS")
+    private String patientRecordStatus;
+    @Column(name = "PATIENT_LOCAL_ID")
+    private String patientLocalId;
+    @Column(name = "PATIENT_AGE_REPORTED")
+    private String patientAgeReported;
+    @Column(name = "PATIENT_AGE_REPORTED_UNIT")
+    private String patientAgeReportedUnit;
+    @Column(name = "PATIENT_CURRENT_SEX")
+    private String patientCurrentSex;
+    @Column(name = "PATIENT_ENTRY_METHOD")
+    private String patientEntryMethod;
+    @Column(name = "PATIENT_LAST_CHANGE_TIME")
+    private String patientLastChangeTime;
+    @Column(name = "PATIENT_FIRST_NAME")
+    private String patientFirstName;
+    @Column(name = "PATIENT_MIDDLE_NAME")
+    private String patientMiddleName;
+    @Column(name = "PATIENT_LAST_NAME")
+    private String patientLastName;
+    @Column(name = "PATIENT_NAME_SUFFIX")
+    private String patientNameSuffix;
+    @Column(name = "PATIENT_NAME")
+    private String patientName;
```

Comment:
These are the json string fields right. May be just add a comment on these fields for reference.

Reply:
Yes. Will differentiate the json fields from the Person fields in the next iteration of sproc/output structure.